example/lib/apis/user_apis.dart

```diff
@@ -5,6 +5,11 @@ callUpdateProfileApi(String name, String mobile) {
       name, mobile, (bool isUpdated) {}, (ApiErrorModel error) {});
 }
 
+shouldEnableNotification(bool notification) {
```

Comment:
change the signature to
enableNotification(bool shouldEnableNotification) {
  UserApiManager.shouldEnableNotification(
      shouldEnableNotification, (bool isEnabled) {}, (ApiErrorModel error) {});
}

Reply:
done